modules/end-user-guide/partials/proc_using-a-web-reference-for-your-ide.adoc

```diff
@@ -0,0 +1,16 @@
+[id="using-a-web-reference-for-your-ide_{context}"]
+=== Using a web reference for your IDE
+
+Rather than pointing to a plug-in in your plug-in registry, you can point at a YAML file directly with the `reference` directive.
+
+.Procedure
+
+* Set a `reference` directive in your `che-editor.yaml` file:
++
```

Comment:
```suggestion
+
.Example
```
If your code samples show just one of multiple possible configurations, it's better to identify them as examples. You can do this by using the `.Example` heading, like I've suggested here. Or, you can simply end the preceding sentence with `. For example:`.